tests/attestation_plugin/enc/enc.c

```diff
@@ -70,18 +70,27 @@ static void _test_sgx_remote()
             0,
             &evidence,
             &evidence_size,
-            NULL,
-            0),
+            &endorsements,
+            &endorsements_size),
```

Comment:
Should keep these two lines as `NULL` and `0`. These values are used to test oe_get_evidence() proper handling of `NULL` endorsements parameters.

Reply:
Oh I see.
So maybe we can keep this `NULL` endorsement test for attester unchanged and add another `NULL` endorsement test for verifier. Sounds good?

Reply:
> Oh I see.
> So maybe we can keep this `NULL` endorsement test for attester unchanged and add another `NULL` endorsement test for verifier. Sounds good?

Sounds good.